src/main/java/frc/robot/commandGroups/MiddleScoring.java

```diff
@@ -21,7 +22,9 @@ public MidScoring(){
                         () -> limelight.getPipeline() == VisionConstants.REFLECTIVE_TAPE_PIPELINE
                 ),
                 new SetArmsPosition(() -> limelight.getPipeline() == VisionConstants.REFLECTIVE_TAPE_PIPELINE ? ArmConstants.MIDDLE_CONE_SCORING : ArmConstants.MIDDLE_CUBE_SCORING),
+                new WaitCommand(3),
                 new InstantCommand(gripper::open),
+                new WaitCommand(3),
```

Comment:
I would create a constant for the timeout, as probably it's going to change. Alsom In the instant command, require the gripper